images/ose-network-tools.yml

```diff
@@ -16,7 +16,8 @@ for_payload: false
 from:
   builder:
   - stream: golang
-  member: openshift-enterprise-base
+  - member: openshift/ose-ovn-kubernetes
```

Comment:
Q: should this be `openshift-enterprise-ovn-kubernetes` and `openshift-enterprise-tools` instead ?

Reply:
looks like it should be `ose-ovn-kubernetes` and `ose-tools`